docs/modules/release-notes/pages/5-4-0.adoc

```diff
@@ -0,0 +1,353 @@
+https://github.com/hazelcast/hazelcast/pull/23509[#23509]
+* Fixed the Javadoc for caches where it was referring incorrectly to statistics instead of management, for the `setManagementEnabled()` method.
+https://github.com/hazelcast/hazelcast/pull/22575[#22575]
+* Implemented RAFT lifecycle interfaces for CPMap. #6800
+* Added the `getCPObjectInfos()` method to create a snapshot of all existing CP objects for a given service on the group leader. #615
+* Added the `getCPGroupIds()` method to the CP Subsystem API to allow listing all data structures within a CP group. #591
+* The "last update time" record of the map entries recovered from the disk persistence is not used anymore when calculating the entries' expiration times. #233
+* [.enterprise]*Enterprise* Added the ability to check whether the `ADVANCED_CP` feature is present in the Hazelcast Platform Enterprise license, to enable CPMap. #157
+
+=== [.enterprise]*Enterprise* WAN Replication
+
+* Improved dead connection handling for WAN replication static IP discovery, by introducing health check to the discovery strategy.
+https://github.com/hazelcast/hazelcast/pull/25364[#25364]
+* Added support for the dynamic adding of WAN replication configurations using Java API.
+https://github.com/hazelcast/hazelcast/pull/25118[#25118]
+* Added support for evicting map and cache entries through WAN replication by introducing two new properties. When enabled, WAN replication
+events are fired when values are evicted from the map and cache objects. See xref:wan:tuning.adoc#replicating-imap-icache-evictions[Replicating `IMap`/`ICache` Evictions].
+https://github.com/hazelcast/hazelcast/pull/24941[#24941]
+
+=== [.enterprise]*Enterprise* Security
+
+* Added the ability to check map permissions before suggesting a schema. If a client has permission to read a map, then a suggestion with schema is sent. Otherwise a warning that no
+permissions to read maps have been set is generated.
+https://github.com/hazelcast/hazelcast/pull/26058[#26058]
+* Updated permission checks in `CacheCreateConfig` and `GetSemaphoreType` tasks.
+https://github.com/hazelcast/hazelcast/pull/25529[#25529]
+* Improved the permission checks in the file connectors by adding a method that returns the permissions required to resolve field names.
+https://github.com/hazelcast/hazelcast/pull/25348[#25348]
+* Added support for permission subtraction (deny permissions) in client connections. See xref:security:native-client-security.adoc#deny-permissions[Deny Permissions].
+https://github.com/hazelcast/hazelcast/pull/25154[#25154]
+* Added the boolean `forceCertValidation` property to the security configuration to initiate a remote certificate validity check. #6235
+
+=== Storage
+
+* Improved the hit/miss cache statistics counter performance.
+https://github.com/hazelcast/hazelcast/pull/25146[#25146]
+* [.enterprise]*Enterprise* Tiered Store can now be used for map loaders.
```

Comment:
```suggestion
* [.enterprise]*Enterprise* Tiered Store can now be used with map loaders.
```

Previously we wouldn't allow a map configured with a map store to be also configured with tiered storage. Now users can enable both tiered storage AND a map store for an IMap.